Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -48,6 +48,10 @@ public final class CardPresentPaymentStore: Store {
                            parameters: parameters,
                            onCardReaderMessage: event,
                            onCompletion: completion)
+        case .checkForUpdate(onData: let data, onCompletion: let completion):
+            checkForUpdate(onData: data, onCompletion: completion)
+        case .update(onProgress: let progress, onCompletion: let completion):
```

Comment:
The names of these actions could be clearer in this context? Perhaps, similar to `startCardReaderDisccovery` and `cancelCardReaderDiscovery` we could have `checkForCardReaderUpdate` and `startCardReaderUpdate`